---
layout: page
title: Yang Group's publications
---

<div class="navbar">
  <div class="navbar-inner">
      <ul class="nav">
          <li><a href="https://scholar.google.com/citations?user=0CK0320AAAAJ">Google Scholar</a></li>
      </ul>
  </div>
</div>

<u>Lab Member</u>; &dagger;: Co-first authors; &sect;: Corresponding authors.

### Preprint
<ol start="2"> <li> Chen, J., Liu, W., Luo, T., …, <b>Yang, Y.</b>, Li, Y. (2022+) A comprehensive comparison on cell type composition inference for spatial transcriptomics data. <i><b>bioRxiv</b></i>. <<a href="https://www.biorxiv.org/content/10.1101/2022.02.20.481171v1">link</a>></span></li></ol>

<ol start="1" style="font-weight: bold;"> <li><span style="font-weight: normal;">Giusti-Rodríguez, P.<sup>&dagger;</sup>, Lu, L.<sup>&dagger;</sup>, <b>Yang, Y.<sup>&dagger;</sup></b>, ..., Sullivan, P. (2022+) Using three-dimensional regulatory chromatin interactions from adult and fetal cortex to interpret genetic results for psychiatric disorders and cognitive traits. <i><b>bioRxiv</b></i>. <<a href="https://www.biorxiv.org/content/10.1101/406330v1.full">link</a>></span></li></ol>

### 2022
<ol start="55" style="font-weight: bold;"> <li><span style="font-weight: normal;"> Huang, L.<sup>&dagger;</sup>, <b>Yang, Y.<sup>&dagger;</sup></b>, Li, G., Jiang, M., Wen, J., Abnousi, A., Rosen, J., Hu, M., Li, Y. (2022) Leveraging deep learning methods developed for Hi-C data to enhance resolution of HiChIP/PLAC-seq data. <i><b>Briefings in Bioinformatics</b></i>. Accepted. <<a href="">link</a>></span></li></ol>

<ol start="54"> <li> Zhao, B., Li, T., Smith, S., ..., <b>Yang, Y.</b>, ..., Zhu, H. (2022) Common variants contribute to intrinsic human brain functional networks. <i><b>Nature Genetics</b></i>. Accepted. <<a href="https://www.biorxiv.org/content/10.1101/2020.07.30.229914v2.full">link</a>></li></ol>

<ol start="53" style="font-weight: bold;"> <li><span style="font-weight: normal;"> Qian, J.<sup>&dagger;</sup>, Wang, N.<sup>&dagger;</sup>, <u>Ren, W.</u><sup>&dagger;</sup>, Zhang, R., <u>Hong, X.</u>, Chen, L., Zhang, K., Shu, Y., Hu, N., <b>Yang, Y.<sup>&sect;</sup></b> (2022) Molecular dissection unveiling dwarfing effects of plant growth retardants on pomegranate. <i><b>Frontiers in Plant Science</b></i>, 13: 866193. PMID: . <<a href="https://www.frontiersin.org/articles/10.3389/fpls.2022.866193/full?utm_source=Email_to_authors_&utm_medium=Email&utm_content=T1_11.5e1_author&utm_campaign=Email_publication&field=&journalName=Frontiers_in_Plant_Science&id=866193">link</a>></span></li></ol>

<ol start="52" style="font-weight: bold;"> <li><span style="font-weight: normal;"> Janghorban, M.<sup>&dagger;</sup>, <b>Yang, Y.<sup>&dagger;</sup></b>, Zhao, N., Hamor, C., Nguyen, T., Zhang, X., Rosen, J. (2022) Single Cell Analysis Unveils the Role of the Tumor Immune Microenvironment and Notch Signaling in Dormant Minimal Residual Disease. <i><b>Cancer Research</b></i>, 82: 885–899. PMID: 34965936. <<a href="https://aacrjournals.org/cancerres/article-abstract/82/5/885/681773/Single-Cell-Analysis-Unveils-the-Role-of-the-Tumor?redirectedFrom=fulltext">link</a>></span></li></ol>

<ol start="51"> <li> Wen, J., Taylor, L., Sun, Q., <b>Yang, Y.</b>, Chen, J., Harigaya, Y., Ulirsch, J., Sankaran, V., Hu, M., Reiner, A., Raffield, L., Li, Y. (2022) Super interactive promoters provide insight into cell type-specific regulatory networks in blood lineage cell types. <i><b>PLOS Genetics</b></i>, 18: e1009984. PMID: 35100265. <<a href="https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1009984">link</a>></li></ol>

<ol start="50"> <li> Xu, S., Guo, Z., Feng, X., Shao, S., <b>Yang, Y.</b>, Li, J., Zhong, C., He, Z., Shi, S. (2022) Where whole-genome duplication is most beneficial - Adaptation of mangroves to a wide salinity range between land and sea. <i><b>Molecular Ecology</b></i>. DOI: 10.1111/mec.16320. PMID: 34882881. <<a href="https://onlinelibrary.wiley.com/doi/10.1111/mec.16320">link</a>></li></ol>

<ol start="49"> <li> Tang, X., Huang Y., Li, H., …, <b>Yang, Y.</b>, Pang, H. (2022) Genomic insight into the scale specialization of the biological control agent Novius pumilus (Weise, 1892). <i><b>BMC Genomics</b></i>, 23: 90. PMID: 35100986. <<a href="https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-022-08299-w">link</a>></li></ol>

<ol start="48" style="font-weight: bold;"> <li><span style="font-weight: normal;"> Yang, Y., Sun, Q., Huang, L., Broome, J., Correa, A., Reiner, A., NHLBI Trans-Omics for Precision Medicine (TOPMed) Consortium, Raffield, L.<sup>&sect;</sup>, <b>Yang, Y.<sup>&sect;</sup></b>, Li, Y.<sup>&sect;</sup> (2022) eSCAN: Scan Regulatory Regions for Aggregate Association Testing using Whole Genome Sequencing Data. <i><b>Briefings in Bioinformatics</b></i>, 23: bbab497. PMID: 34882196. <<a href="https://academic.oup.com/bib/advance-article/doi/10.1093/bib/bbab497/6457165?searchresult=1">link</a>></span></li></ol>

<ol start="47" style="font-weight: bold;"> <li><span style="font-weight: normal;"> Wang, H.<sup>&dagger;</sup>, <b>Yang, Y.<sup>&dagger;</sup></b>, Qian, Y., Liu, J., Qian, L. (2022) Delineating Chromatin Accessibility Re-patterning at Single Cell Level during Early Stage of Direct Cardiac Reprogramming. <i><b>Journal of Molecular and Cellular Cardiology</b></i>, 162: 62-71. PMID: 34509499. <<a href="https://www.sciencedirect.com/science/article/pii/S0022282821001747">link</a>></span></li></ol>

<ol start="46" style="font-weight: bold;"> <li><span style="font-weight: normal;"> Wang, L.<sup>&dagger;</sup>, <b>Yang, Y.<sup>&dagger;</sup></b>, Ma, H., ...., Liu, J., Qian, L. (2022) Single Cell Dual-Omics Reveals the Transcriptomic and Epigenomic Diversity of Cardiac Non-myocytes. <i><b>Cardiovascular Research</b></i>, cvab134. PMID: 33839759. <<a href="https://academic.oup.com/cardiovascres/advance-article-abstract/doi/10.1093/cvr/cvab134/6220323">link</a>></span></li></ol>

### 2021

<ol start="45"> <li> Rosen, J., <b>Yang, Y.</b>, Abnousi, A., Chen, J., Song, M., Jones, I., Shen, Y., Hu, M., Li, Y. (2021) HPRep: Quantifying Reproducibility in HiChIP and PLAC-seq Datasets. <i><b>Current Issues in Molecular Biology</b></i>, 43: 1156-1170. PMID: 34563051. <<a href="https://www.mdpi.com/1467-3045/43/2/82">link</a>></li></ol>

<ol start="44" style="font-weight: bold;"> <li><span style="font-weight: normal;"> Ma, H.<sup>&dagger;</sup>, Liu, Z.<sup>&dagger;</sup>, <b>Yang, Y.<sup>&dagger;</sup></b>, ..., Liu, J. (2021) Coordinated transcriptome and cell state dynamics of non-myocytes in heart regeneration. <i><b>EMBO reports</b></i>, 22: e52901. PMID: 34523214. <<a href="https://www.embopress.org/doi/epdf/10.15252/embr.202152901">link</a>></span></li></ol>

<ol start="43" style="font-weight: bold;"> <li><span style="font-weight: normal;"> Li, G., Luan, C., Dong, Y., …, Li, Y.<sup>&sect;</sup>, <b>Yang, Y.<sup>&sect;</sup></b> (2021) ExpressHeart: Web Portal to Visualize Transcriptome Profiles of Non-cardiomyocyte Cells. <i><b>International Journal of Molecular Sciences</b></i>, 22: 8943. PMID: 34445647. <<a href="https://www.mdpi.com/1422-0067/22/16/8943">link</a>></span></li></ol>

<ol start="42"> <li> Yang, Y., Sun, H., Zhang, Y., …, <b>Yang, Y.</b>, Wu, D., Yu, D. (2021) Dimensionality reduction by UMAP reinforces sample heterogeneity analysis in bulk transcriptomic data. <i><b>Cell Reports</b></i>, 36: 109442. PMID: 34320340. <<a href="https://www.sciencedirect.com/science/article/pii/S2211124721008597">link</a>></li></ol>

<ol start="41"> <li> Yu, M., Abnousi, A., Zhang, Y., ..., <b>Yang, Y.</b>, ..., Ren, B., Hu, M. (2021) SnapHiC: a computational pipeline to identify chromatin loops from single cell Hi-C data. <i><b>Nature Methods</b></i>, 18: 1056–1059. PMID: 34446921. <<a href="https://www.nature.com/articles/s41592-021-01231-2">link</a>></li></ol>

<ol start="40" style="font-weight: bold;"> <li><span style="font-weight: normal;"> Liu, W.<sup>&dagger;</sup>, <b>Yang, Y.<sup>&dagger;</sup></b>, Abnousi, A., Zhang, Q., Kubo, N., Martin Beem, J., Li, Y., Hu, M. (2021) MUNIn: a statistical framework for identifying long-range chromatin interactions from multiple samples. <i><b>Human Genetics and Genomics Advances</b></i>, 2: 100036. PMID: 34485947. <<a href="https://www.sciencedirect.com/science/article/pii/S2666247721000178?via%3Dihub">link</a>></span></li></ol>

<ol start="39" style="font-weight: bold;"> <li><span style="font-weight: normal;"><b> Yang, Y.</b>, Li, G., Xie, Y., Wang, L., Lagler, T., Yang, Y., Liu, J., Qian, L., Li, Y. (2021) iSMNN: Batch Effect Correction for Single-cell RNA-seq data via Iterative Supervised Mutual Nearest Neighbor Refinement. <i><b>Briefings in Bioinformatics</b></i>, 22: bbab122. PMID: 33839756. <<a href="https://academic.oup.com/bib/article-abstract/22/5/bbab122/6220171?redirectedFrom=fulltext">link</a>></span></li></ol>

<ol start="38"> <li> Yu, C., Littleton S., Giroux, N., ..., <b>Yang, Y.</b>, ..., Shen, X., Saban, D. (2021) Mucosal Associated Invariant T (MAIT) Cell Responses Differ by Sex in COVID-19. <i><b>Med</b></i>, 2: 755-772. PMID: 33870241. <<a href="https://www.cell.com/med/pdf/S2666-6340(21)00157-4.pdf?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS2666634021001574%3Fshowall%3Dtrue">link</a>></li></ol>

<ol start="37"> <li> Jiang, Y., Li, W., Lindsey-Boltz, L., <b>Yang, Y.</b>, Li, Y., Sancar, A. (2021) Super-hotspots and -coldspots in the repair of UV-induced DNA damage in the human genome. <i><b>Journal of Biological Chemistry</b></i>, 296: 100581. PMID: 33771559. <<a href="https://www.jbc.org/article/S0021-9258(21)00361-6/abstract">link</a>></li></ol>

<ol start="36"> <li> Jiang, W., <b>Yang, Y.</b>, Mercer-Smith, A., ..., Hingtgen S. (2021) Development of next-generation tumor-homing induced neural stem cells to enhance treatment of metastatic cancers. <i><b>Science Advances</b></i>, 7: eabf1526. PMID: 34108203. <<a href="https://advances.sciencemag.org/content/7/24/eabf1526">link</a>></li></ol>

<ol start="35"> <li> Wang, H., <b>Yang, Y.</b>, Liu, J., Qian, L. (2021) Direct cell reprogramming: approaches, mechanisms and progress. <i><b>Nature Reviews Molecular Cell Biology</b></i>, 22: 410–424. PMID: 33619373. <<a href="https://www.nature.com/articles/s41580-021-00335-z#citeas">link</a>></li></ol>

<ol start="34"> <li> Bruinsma, R., Fajgenbaum, K., <b>Yang, Y.</b>, del Mar Melendez-Gonzalez, M., Mohlke, K., Li, Y., Sayed, C. (2021) Assessment of Familial Risk in Patients with Hidradenitis Suppurativa. <i><b>British Journal of Dermatology</b></i>, 184: 753-754. PMID: 33152101. <<a href ="https://onlinelibrary.wiley.com/doi/10.1111/bjd.19664">link</a>></li></ol>

<ol start="33" style="font-weight: bold;"> <li><span style="font-weight: normal;"> Lagler, T., Abnousi, A., Hu, M., <b>Yang, Y.<sup>&sect;</sup></b>, Li, Y.<sup>&sect;</sup> (2021) HiC-ACT: improved chromatin interactions from Hi-C data via aggregated Cauchy test. <i><b>American Journal of Human Genetics</b></i>, 108: 257-268. PMID: 33545029. <<a href="https://www.sciencedirect.com/science/article/pii/S0002929721000094?dgcid=coauthor">link</a>></span></li></ol>

<ol start="32" style="font-weight: bold;"> <li><span style="font-weight: normal;"> Crowley, C., <b>Yang, Y.<sup>&sect;</sup></b>, Qiu, Y., ..., Hu, M.<sup>&sect;</sup>, Li, Y.<sup>&sect;</sup> (2021) FIREcaller: Detecting Frequently Interacting Regions from Hi-C Data. <i><b>Computational and Structural Biotechnology Journal</b></i>, 19: 355-362. PMID: 33489005. <<a href="https://www.sciencedirect.com/science/article/pii/S2001037020305511?via%3Dihub">link</a>></span></li></ol>

<ol start="31" style="font-weight: bold;"> <li><span style="font-weight: normal;"><b>Yang, Y.<sup>&dagger;</sup></b>, Li, G.<sup>&dagger;</sup>, Qian, H., Wilhelmsen, K., Shen, Y., Li., Y. (2021) SMNN: Batch Effect Correction for Single-cell RNA-seq data via Supervised Mutual Nearest Neighbor Detection. <i><b>Briefings in Bioinformatics</b></i>, 22: bbaa097. PMID: 32591778. <<a href="https://academic.oup.com/bib/article-abstract/22/3/bbaa097/5855265?redirectedFrom=fulltext">link</a>></span></li></ol>

### 2020

<ol start="30" style="font-weight: bold;"> <li><span style="font-weight: normal;"> Qian, J., Zhang, X., Yan, Y., Wang, N., Ge, W., Zhou, Q.<sup>&sect;</sup>, <b>Yang, Y.<sup>&sect;</sup></b> (2020) Unravelling the molecular mechanisms of Abscisic acid-mediated drought-stress alleviation in pomegranate (<i>Punica granatum</i> L.). <i><b>Plant Physiology and Biochemistry</b></i>, 157: 211-218. PMID: 33120112. <<a href="https://www.sciencedirect.com/science/article/pii/S0981942820305222">link</a>></span></li></ol>

<ol start="29"> <li> Wang, L., Ma, H., Huang, P., ..., <b>Yang, Y.</b>, ..., Qian, L. (2020) Down-regulation of Beclin1 promotes direct cardiac reprogramming. <i><b>Science Translational Medicine</b></i>, 12(566): eaay7856. PMID: 33087505. <<a href="https://stm.sciencemag.org/content/12/566/eaay7856/tab-pdf">link</a>></li></ol>

<ol start="28" style="font-weight: bold;"> <li><span style="font-weight: normal;"><b>Yang, Y.</b>, Li, Y., Sancar, A., Oztas, O. (2020) The circadian clock shapes the Arabidopsis transcriptome through regulating alternative splicing and alternative polyadenylation. <i><b>Journal of Biological Chemistry</b></i>, 295: 7608-7619. PMID: 32303634. <<a href="https://www-jbc-org.libproxy.lib.unc.edu/content/295/22/7608">link</a>></span></li></ol>

<ol start="27"> <li> Feng, X., Xu, S., Li, J., <b>Yang, Y.</b>, Chen, Q., Lyu, H., Zhong, C., He, Z., Shi, S. (2020) Molecular adaptation to salinity fluctuation in tropical intertidal environments of a mangrove tree <i>Sonneratia alba</i>. <i><b>BMC Plant Biology</b></i>, 20: 178. PMID: 32321423. <<a href="https://bmcplantbiol.biomedcentral.com/articles/10.1186/s12870-020-02395-3">link</a>></li></ol>

<ol start="26"> <li>Johnson, N., Huang, L., Li, R., ..., <b>Yang, Y.</b>, ..., Conneely, K. (2020) Age-related DNA hydroxymethylation is enriched for gene expression and immune system processes in human peripheral blood. <i><b>Epigenetics</b></i>, 15: 294-306. PMID: 31506003. <<a href="https://www.tandfonline.com/doi/abs/10.1080/15592294.2019.1666651">link</a>></li></ol>

<ol start="25" style="font-weight: bold;"> <li><span style="font-weight: normal;">Huh, R.<sup>&dagger;</sup>, <b>Yang, Y.<sup>&dagger;</sup></b>, Jiang, Y., Shen, Y., Li, Y. (2020) SAME-clustering: Single-cell Aggregated Clustering via Mixture Model Ensemble. <i><b>Nucleic Acids Research</b></i>, 48: 86-95. PMID: 31777938. <<a href="https://academic.oup.com/nar/article/48/1/86/5644992">link</a>></span></li></ol>


### 2019

<ol start="24"> <li>Li, G., <b>Yang, Y.</b>, Van Buren, E., Li, Y. (2019) Dropout Imputation and Batch Effect Correction for Single-Cell RNA-seq Data. <i><b>Journal of Bio-X Research</b></i>, 2: 169-177. <<a href="https://journals.lww.com/jbioxresearch/Fulltext/2019/12000/Dropout_imputation_and_batch_effect_correction_for.4.aspx">link</a>></li></ol>

<ol start="23"> <li>Giusti-Rodriguez, P., Lu, L, <b>Yang, Y.</b>, ..., Sullivan, P. (2019) SA72A HIGH-RESOLUTION MAP OF CHROMATIN INTERACTIONS IN ADULT AND FETAL CORTEX. <i><b>European Neuropsychopharmacology</b></i>, 29: S1227. <<a href="https://www.sciencedirect.com/science/article/abs/pii/S0924977X18305984?via%3Dihub">link</a>></li></ol>

<ol start="22" style="font-weight: bold;"> <li><span style="font-weight: normal;"><b>Yang, Y.</b>, Huh, R., Culpepper, H., Lin, Y., Love, M., Li, Y. (2019) SAFE-clustering: Single-cell Aggregated (From Ensemble) Clustering for Single-cell RNA-seq Data. <i><b>Bioinformatics</b></i>, 35: 1269-1277. PMID: 30202935. <<a href="https://academic.oup.com/bioinformatics/article-abstract/35/8/1269/5092931">link</a>></span></li></ol>

<ol start="21" style="font-weight: bold;"> <li><span style="font-weight: normal;">Zhou Q.<sup>&dagger;</sup>, <b>Yang Y.<sup>&dagger;</sup></b>, Yang Z. (2019) Molecular dissection of cadmium-responsive transcriptome profile in a low-cadmium-accumulating cultivar of <i>Brassica parachinensis</i>. <i><b>Ecotoxicology and Environmental Safety</b></i>, 176: 85-94. PMID: 30921700. <<a href="https://www.sciencedirect.com/science/article/abs/pii/S0147651319303495">link</a>></span></li></ol>

<ol start="20"> <li>Juric I., Yu M., Abnousi A., ..., <b>Yang Y.</b>, ..., Bing Ren B., Hu M. (2019) MAPS: model-based analysis of long-range chromatin interactions from PLAC-seq and HiChIP experiments. <i><b>PLOS Computational Biology</b></i>, 15(4): e1006982. PMID: 30986246. <<a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1006982">link</a>></li></ol>

### 2018

<ol start="19" style="font-weight: bold;"> <li><span style="font-weight: normal;"><b>Yang, Y.</b>, Guo, W., Shen, X., Li, J., Yang, S., Chen, S., He, Z., Zhou R., Shi, S. (2018) Identification and characterization of evolutionarily conserved alternative splicing events in a mangrove genus <i>Sonneratia</i>. <i><b>Scientific Reports</b></i>, 8: 4425. PMID: 29535339. <<a href="https://www.nature.com/articles/s41598-018-22406-6">link</a>></span></li></ol>

<ol start="18"> <li>Guo, Z., Li, X., He, Z., <b>Yang, Y.</b>, Wang, W., Zhong, C., Greenberg, A.J., Wu, C-I, Duke N.C., Shi, S. (2018) Extremely low genetic diversity across mangrove taxa reflects past sea level changes and hints at poor future responses. <i><b>Global Change Biology</b></i>, 24: 1741–1748. PMID: 29087007. <<a href="https://onlinelibrary.wiley.com/doi/abs/10.1111/gcb.13968">link</a>></li></ol>

### 2017

<ol start="17" style="font-weight: bold;"> <li><span style="font-weight: normal;"><b>Yang, Y.</b>, Li, J., Yang, S., Li, X., Fang, L., Zhong, C., Duke, N., Zhou, R., Shi, S. (2017) Effects of Pleistocene sea-level fluctuations on mangrove population dynamics: A lesson from <i>Sonneratia alba</i>. <i><b>BMC Evolutionary Biology</b></i>, 17: 22. PMID: 28100168. <<a href="https://link.springer.com/article/10.1186/s12862-016-0849-z">link</a>></span></li></ol>

<ol start="16"> <li>Li, J., <b>Yang, Y.</b>, Yang, S., Zhang, Z., Chen, S., Zhong, C., Zhou, R., Shi, S. (2017) Comparative transcriptome analyses of a mangrove tree <i>Sonneratia caseolaris</i> and its non-mangrove relatives, <i>Trapa bispinosa</i> and <i>Duabanga grandiflora</i>. <i><b>Marine Genomics</b></i>, 31: 13-15. PMID: 27810366. <<a href="https://www.sciencedirect.com/science/article/abs/pii/S1874778716301349">link</a>></li></ol>

<ol start="15"> <li>Zhou, Q., <b>Yang, Y.</b>, Shen, C., He, C., Yuan, J., Yang, Z. (2017) Comparative analysis between low- and high-cadmium-accumulating cultivars of <i>Brassica parachinensis</i> to identify difference of cadmium-induced microRNA and their targets. <i><b>Plant and Soil</b></i>, 420: 223-237. <<a href="https://link.springer.com/article/10.1007/s11104-017-3380-0">link</a>></li></ol>

<ol start="14"> <li>Du, Y., Martin, J., McGee, J., <b>Yang, Y.</b>, Liu, E., Sun, Y., Geihs, M., Kong, X., Zhou, E., Li, Y., Huang, J. (2017) A SNP panel and online tool for checking genotype concordance through comparing QR codes. <i><b>PLoS ONE</b></i>, 12(9): e0182438. PMID: 28926565. <<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5604942/">link</a>></li></ol>

### 2016

<ol start="13" style="font-weight: bold;"> <li><span style="font-weight: normal;"><b>Yang, Y.</b>, Duke, N.C., Peng, F., Li, J., Yang, S., Zhong, C., Zhou, R., Shi, S. (2016) Ancient geographical barriers drive differentiation among <i>Sonneratia caseolaris</i>. populations and recent divergence from <i>S. lanceolata</i>. <i><b>Frontiers in Plant Science</b></i>, 7: 1618. PMID: 27833634. <<a href="https://www.frontiersin.org/articles/10.3389/fpls.2016.01618/full">link</a>></span></li></ol>

<ol start="12"> <li>Li, J., <b>Yang, Y.</b>, Chen, Q., ..., Zhou R., Shi. S. (2016) Pronounced genetic differentiation and recent secondary contact in the mangrove tree <i>Lumnitzera racemosa</i> revealed by population genomic analyses. <i><b>Scientific Reports</b></i>, 6: 29486. PMID: 27380895. <<a href="https://www.nature.com/articles/srep29486">link</a>></li></ol>

<ol start="11"> <li>Fang, L., <b>Yang, Y.</b>, Guo, W., Li, J., Zhong, C., Huang, Y., Zhou, R., Shi, S. (2016) De novo assembly of the transcriptome of <i>Aegiceras corniculatum</i>, a mangrove species in the Indo-West Pacific region. <i><b>Marine Genomics</b></i>, 28: 49–52. PMID: 27296448. <<a href="https://www.sciencedirect.com/science/article/abs/pii/S1874778716300526">link</a>></li></ol>

<ol start="10"> <li>Li, X., Duke, N., <b>Yang, Y.</b>, Huang, L., Zhu, Y., Zhang, Z., Zhou, R., Zhong, C., Huang, Y., Shi, S. (2016) Re-Evaluation of Phylogenetic Relationships among Species of the Mangrove Genus <i>Avicennia</i> from Indo-West Pacific Based on Multilocus Analyses. <i><b>PLoS ONE</b></i>, 11(10): e0164453. PMID: 27716800. <<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5055292/">link</a>></li></ol>

<ol start="9"> <li>Wang, S., Chen, Y., <b>Yang, Y.</b>, Wu, W., Liu, Y., Fan, Q., Zhou, R. (2016) Phylogenetic relationships and natural hybridization in <i>Triadica</i> inferred from nuclear and chloroplast DNA analyses. <i><b>Biochemical Systematics and Ecology</b></i>, 64: 142-148. <<a href="https://www.sciencedirect.com/science/article/pii/S0305197815002938">link</a>></li></ol>

<ol start="8"> <li>Zhang, Z., He, Z., Xu, S., Li, X., Guo, W., <b>Yang, Y.</b>, Zhong, C., Zhou, R., Shi, S. (2016) Transcriptome analyses provide insights into the phylogeny and adaptive evolution of the mangrove fern genus <i>Acrostichum</i>. <i><b>Scientific Reports</b></i>, 6: 35634. PMID: 27782130. <<a href="https://www.nature.com/articles/srep35634">link</a>></li></ol>

<ol start="7"> <li>Shi, L., Li, N., Wang, S., Zhou, Y., Huang, W., <b>Yang, Y.</b>, Ma, Y., Zhou, R. (2016) Molecular Evidence for the Hybrid Origin of <i>Ilex dabieshanensis</i> (Aquifoliaceae). <i><b>PLoS ONE</b></i>, 11(1): e0147825. PMID: 26808531. <<a href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0147825">link</a>></li></ol>

### 2015

<ol start="6" style="font-weight: bold;"> <li><span style="font-weight: normal;"><b>Yang Y.</b>, Yang, S., Li, J., Deng, Y., Zhang, Z., Xu, S., Guo, W., Zhong, C., Zhou, R., Shi, S. (2015) Transcriptome analysis of the Holly mangrove <i>Acanthus ilicifolius</i> and its terrestrial relative, <i>Acanthus leucostachyus</i>, provides insights into adaptation to intertidal zones. <i><b>BMC Genomics</b></i>, 16: 605. PMID: 26272068. <<a href="https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-015-1813-9">link</a>></span></li></ol>

<ol start="5" style="font-weight: bold;"> <li><span style="font-weight: normal;"><b>Yang, Y.</b>, Yang, S., Li, J., Li, X., Zhong, C., Huang, Y., Zhou, R., Shi, S. (2015) De novo assembly of the transcriptomes of two yellow mangroves, <i>Ceriops tagal</i>  and <i>C. zippeliana</i> , and one of their terrestrial relatives, <i>Pellacalyx yunnanensis</i>. <i><b>Marine Genomics</b></i>, 23: 33-36. PMID: 25899405. <<a href="https://www.sciencedirect.com/science/article/abs/pii/S1874778715000586">link</a>></span></li></ol>

<ol start="4" style="font-weight: bold;"> <li><span style="font-weight: normal;"><b>Yang, Y.</b>, Yang, S., Fang, L., Li, J., Zhong, C., Zhou, R., Shi, S. (2015) Phylogenetic position of <i>Sonneratia  griffithii</i>  based on sequences of the nuclear ribosomal internal transcribed spacer and 13 nuclear genes. <i><b>Journal of Systematics and Evolution</b></i>, 53(1): 47-52. <<a href="https://onlinelibrary.wiley.com/doi/abs/10.1111/jse.12102">link</a>></span></li></ol>

### 2014

<ol start="3" style="font-weight: bold;"> <li><span style="font-weight: normal;"><b>杨宇晨</b>, 李建芳, 郭无瑕, 钟才荣, 周仁超, 施苏华. (2014) 植物物种形成的模式与机制. <i><b>生命科学</b></i>, 26: 138-143.</span></li></ol>

### 2011

<ol start="2"> <li>钟才荣, 李诗川, <b>杨宇晨</b>, 张影, 林作武. (2011) 红树植物拉关木的引种效果调查研究. <i><b>福建林业科技</b></i>, 38: 96-99.</li></ol>

<ol start="1" style="font-weight: bold;"> <li><span style="font-weight: normal;"><b>杨宇晨</b>, 施苏华. (2011) 亟待保护的中国红树林. <i><b>生命世界</b></i>, 265: 84-87.</span></li></ol>
